[HW] 1072947

unoblesser opened the pull request:
Signed-off-by: 1072947 <s1072947@mail.ncyu.edu.tw>

jrjang commented:
Hi @unoblesser ,

commit message 格式有誤
1. commit summary 為 [HW] 學號
2. commit message 為解決問題與方法

unoblesser commented:
commit message 格式修改，勞煩老師過目

jrjang commented:
Hi @unoblesser ,

commit message 與實際修改不符. 麻煩再確認

Commit:
[HW] 1072947

要解決問題:
由於只能有一筆commit，所以當我們bash的結果錯誤，重新操作完整個流程再次bash，便會出現Your commit number is more than 1
如何解決:
可以到使用者資料夾將整個ncyu-2021-ex1檔案刪除，重新操作完整個流程再次bash即可

Signed-off-by: 1072947 <s1072947@mail.ncyu.edu.tw>
